src/engine-actions/create-channel.js

```diff
@@ -65,7 +65,7 @@ async function createChannel (paymentChannelNetworkAddress, fundingAmount) {
   // because we would fail to fund a channel. This also prevents against having negative
   // values for a fundingAmount after a fee estimate has been taken
   if (balanceCantCoverFees && Big(feeEstimate).gt(fundingAmount)) {
-    throw new Error('fundingAmount does not cover estimated fees', { fundingAmount, feeEstimate })
+    throw new Error('Requested fundingAmount plus fees are larger than uncommitted balance', { fundingAmount, feeEstimate })
```

Comment:
I'm not sure that having an object as the second parameter to an `Error` does anything - have you tested what the output of this looks like?